repos-github.md

```diff
@@ -745,6 +745,13 @@
 - scalabin-no/http4s-directives
 - scalacenter/sbt-scalafix
 - scalacenter/scalafix
+- scalameta/mdoc
+- scalameta/metabrowse
+- scalameta/metals
+- scalameta/munit
+- scalameta/sbt-scalafmt
```

Comment:
cc/ @poslegm @kitbellew what do you think about enabling the Scalafmt repos here? Scala Steward now supports configuring the frequency of pull requests so that you can limit them to only come bi-weekly or monthly, example https://github.com/scalameta/metals/pull/1558

Reply:
@olafurpg is this the process that automatically proposed updates to dependencies of a given repository? that was probably quite handy.

Reply:
I fully support enabling scalafmt repos

Reply:
Correct, this enables the bot that sends PRs to update dependencies. If the volume of PRs is too high you can slow it down by adding a `.scalasteward.conf` file like here 
https://github.com/scalameta/mdoc/blob/57c3b33825924681d22a94c14da963ce71e360c7/.scalasteward.conf#L1-L5

Reply:
Small correction: that file needs to be called `.scala-steward.conf`.

Reply:
Important correction 😅 thanks!

Reply:
Whoops 😬